app/actions/websocket/posts.ts

```diff
@@ -269,6 +269,12 @@ export async function handlePostDeleted(serverUrl: string, msg: WebSocketMessage
             models.push(deleteModel);
         }
 
+        // Check if the deleted post had files to refresh channel stats
+        const hadFiles = (oldPost.metadata?.files?.length || 0) > 0;
+        if (hadFiles) {
+            fetchChannelStats(serverUrl, post.channel_id);
+        }
```

Comment:
Shouldn't we check the stats any time the number of files change? Or is that covered somewhere else?

Reply:
Yes, we should check for the WebSocket when the post is edited as well. I will update the code.